qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -88,6 +88,13 @@ def fontFamily(self, which):
     def defaultTextEncoding(self):
         return self._settings[0].defaultTextEncoding()
 
+    def unknownUrlSchemePolicy(self):
+        return self._settings[0].unknownUrlSchemePolicy()
+
+    def setUnknownUrlSchemePolicy(self, policy):
```

Comment:
Nitpick: For consistency with the other methods/setters, this should be moved below `setDefaultTextEncoding` rather than here.